pants_plugins/sendwave/pants_docker/BUILD

```diff
@@ -1,19 +1,24 @@
-python_library(
+python_sources(
     name="pants_docker_library",
-    dependencies=["pants_plugins:pants",],
+    dependencies=[
+        "pants_plugins:pants",
+    ],
 )
 
 
 python_distribution(
     name="pants_docker",
-    dependencies=[":pants_docker_library",],
+    dependencies=[
+        ":pants_docker_library",
+    ],
     provides=setup_py(
         name="sendwave-pants-docker",
-        version="0.05",
+        version="1.0",
         description="Pants Plugin to automatically generate docker images from pants targets",
         author="compyman@compyman.net",
     ),
-    setup_py_commands=["sdist", "bdist_wheel"],
+    sdist=True,
```

Comment:
These flags are now supported as top-level kwargs:
* https://www.pantsbuild.org/docs/reference-python_distribution#codesdistcode 
* https://www.pantsbuild.org/docs/reference-python_distribution#codewheelcode

Reply:
I think you need to add yourself to the author string too :)

Reply:
Haha! Maybe someday, for now I'd rather someone yell at you instead of me if they have problems with the package 😉